src/main/java/frc/misc2020/FunctionRunnerSendable.java

```diff
@@ -0,0 +1,44 @@
+package frc.misc2020;
+
+import edu.wpi.first.wpilibj.Sendable;
+import edu.wpi.first.wpilibj.SendableBase;
+import edu.wpi.first.wpilibj.smartdashboard.SendableBuilder;
+
+/**
+ * Use this class to have a button on the SmartDashboard that will run a
+ * function.
+ * 
+ * @author schuyler
+ *
+ */
+public class FunctionRunnerSendable extends SendableBase implements Sendable {
```

Comment:
This still needs to be updated to use SendableRegistry instead of SendableBase.

Reply:
Can a student be assigned to work on this?

Reply:
This needs to be fixed before I will approve.